include/compartment.h

```diff
@@ -127,11 +129,12 @@ struct LibDependency
     // TLS-related variables
     // TODO can there be more TLS sections?
     void *tls_sec_addr;
+    ptrdiff_t tls_sec_off;
```

Comment:
Do you want this to be both +ve and -ve?

Reply:
I think this can only be positive. Are you suggesting I should make this `size_t` or some other unsigned type?

Reply:
Actually, checking the source, this is originally a `Elf64_Off`, which `typedef`s to `uint64_t`. So I think `size_t` might be appropriate here, or maybe `uintptr_t`. I have changed this to `size_t` in 77a19fd.